### DISCUSSION TITLE
PwnFox

### DISCUSSION THREAD

**Discussion by rt-bast** [community/community#110320] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi BitK,
I have made some improvements into the PwnFox Plugin (Firefox code to mark only PwnFox containers, tested with burp 2023.12).
You can see the code here: https://github.com/tdf-discipline-cybersecurity/PwnFox

Are you interested to merge this code and republish the plugin?

**Comment by queenofcorgis** [comment]
If you're asking how to use or interact with the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
